contracts/libraries/range/RangePositions.sol

```diff
@@ -165,8 +165,6 @@ library RangePositions {
             cache.amount1
         );
         if (cache.position.liquidity == 0) {
-            cache.position.feeGrowthInside0Last = 0;
-            cache.position.feeGrowthInside1Last = 0;
             cache.position.lower = 0;
```

Comment:
We don't want to clear fee growth values so we can check them after the position is 100% burned in the staking contract.